callautomation-outboundcalling/main.py

```diff
@@ -23,34 +25,60 @@
+# Prompts for text to speech
+SpeechToTextVoice = "en-US-NancyNeural"
+MainMenu = "Hello this is Contoso Bank, we’re calling in regard to your appointment tomorrow at 9am to open a new account. Please confirm if this time is still suitable for you or if you would like to cancel. This call is recorded for quality purposes."
+ConfirmedText = "Thank you for confirming your appointment tomorrow at 9am, we look forward to meeting with you."
+CancelText = "Your appointment tomorrow at 9am has been cancelled. Please call the bank directly if you would like to rebook for another date and time."
+CustomerQueryTimeout = "I’m sorry I didn’t receive a response, please try again."
+NoResponse = "I didn't receive an input, we will go ahead and confirm your appointment. Goodbye"
+InvalidAudio = "I’m sorry, I didn’t understand your response, please try again."
+ConfirmChoiceLabel = "Confirm"
+CancelChoiceLabel = "Cancel"
+RetryContext = "retry"
 
 call_automation_client = CallAutomationClient.from_connection_string(ACS_CONNECTION_STRING)
 recording_id = None
 recording_chunks_location = []
 
 app = Flask(__name__,
-            static_folder=AUDIO_FILES_PATH.strip("/"),
-            static_url_path=AUDIO_FILES_PATH,
             template_folder=TEMPLATE_FILES_PATH)
 
+def get_choices():
+    choices = [
+                RecognitionChoice(label = ConfirmChoiceLabel, phrases= ["Confirm", "First", "One"], tone = DtmfTone.ONE),
+                RecognitionChoice(label = CancelChoiceLabel, phrases= ["Cancel", "Second", "Two"], tone = DtmfTone.TWO)
+            ]
+    return choices
+
+def get_media_recognize_choice_options(call_connection_client: CallConnectionClient, text_to_play: str, target_participant:str, choices: any, context: str):
+     play_source =  TextSource (text= text_to_play, voice_name= SpeechToTextVoice)
+     call_connection_client.start_recognizing_media(
+                input_type=RecognizeInputType.CHOICES,
+                target_participant=target_participant,
+                choices=choices,
+                play_prompt=play_source,
+                interrupt_prompt=False,
+                initial_silence_timeout=10,
+                operation_context=context
+            )
+     
+def handle_play(call_connection_client: CallConnectionClient, text_to_play: str):
+        play_source = TextSource(text=text_to_play, voice_name=SpeechToTextVoice) 
+        call_connection_client.play_media_to_all(play_source)
 
 # GET endpoint to place phone call
 @app.route('/outboundCall')
 def outbound_call_handler():
     target_participant = PhoneNumberIdentifier(TARGET_PHONE_NUMBER)
     source_caller = PhoneNumberIdentifier(ACS_PHONE_NUMBER)
     call_invite = CallInvite(target=target_participant, source_caller_id_number=source_caller)
-    call_connection_properties = call_automation_client.create_call(call_invite, CALLBACK_EVENTS_URI)
+    call_connection_properties = call_automation_client.create_call(call_invite, CALLBACK_EVENTS_URI,
```

Comment:
We have deprecated CallInvite class in 1.1.0
call_connection_properties = call_automation_client.create_call(target_participant, CALLBACK_EVENTS_URI,
                                                                    cognitive_services_endpoint=COGNITIVE_SERVICES_ENDPOINT,
                                                                    source_caller_id_number=source_caller)